sdks/go/container/boot.go

```diff
@@ -46,29 +46,42 @@ func main() {
 	if *id == "" {
 		log.Fatal("No id provided.")
 	}
+	if *provisionEndpoint == "" {
+		log.Fatal("No provision endpoint provided.")
+	}
+
+	ctx := grpcx.WriteWorkerID(context.Background(), *id)
+
+	info, err := provision.Info(ctx, *provisionEndpoint)
+	if err != nil {
+		log.Fatalf("Failed to obtain provisioning information: %v", err)
+	}
+	log.Printf("Provision info:\n%v", info)
+
+	// TODO(BEAM-8201): Simplify once flags are no longer used.
+	if info.GetLoggingEndpoint().GetUrl() != "" {
+		*loggingEndpoint = info.GetLoggingEndpoint().GetUrl()
```

Comment:
Shall we add default values for all the ports so that we avoid passing them whole together in future.

Reply:
Never mind, I overlooked the empty default.